### PR TITLE
Clarify behaviour of rm and down

### PR DESCRIPTION
**Pull request by aanand**
Addresses problems described in #2838

### CODE REVIEW

File: compose/cli/main.py [hunk 1]
```diff
@@ -264,18 +264,29 @@ def create(self, options):
...
         """
-        Stop containers and remove containers, networks, volumes, and images
-        created by `up`. Only containers and networks are removed by default.
+        Stops containers and removes containers, networks, volumes, and images
+        created by `up`.
+
+        By default, the only things removed are:
+
+        - Containers for services defined in the Compose file
+        - Anonymous volumes attached to those containers
```
Comment (dnephin) [inline compose/cli/main.py]: Technically these are not removed, the volume is still there.

File: compose/cli/main.py [hunk 2]
```diff
@@ -264,18 +264,29 @@ def create(self, options):
...
-                                all images, or 'local' to remove only images that
-                                don't have an custom name set by the `image` field
-            -v, --volumes       Remove data volumes
-            --remove-orphans    Remove containers for services not defined in
-                                the Compose file
+            --rmi type          Remove images. Type must be one of:
+                                'all': Remove all images used by any service.
+                                'local': Remove only images that don't have a custom tag
+                                set by the 'image' field.
+            -v, --volumes       Remove named volumes declared in the `volumes` section
```
Comment (dnephin) [inline compose/cli/main.py]: This also removes container volumes like `rm -v`

File: compose/cli/main.py [hunk 3]
```diff
@@ -264,18 +264,29 @@ def create(self, options):
...
+                                of the Compose file.
```
Comment (holgerreif) [inline compose/cli/main.py]: Add info as per [comment](https://github.com/docker/compose/issues/2838#issuecomment-181569292):
Note: compose cannot remove volumes after a call of down w/o specifing -v flag.

**Comment by aanand** [comment]
@dnephin addressed your comments

**Comment by dnephin** [comment]
LGTM

**Pull request merged**